src/main/java/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import model.FruitTransaction;
+import service.FruitTransactionParser;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    @Override
+    public FruitTransaction parse(String line) {
+        String[] parts = line.split(",");
+        FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);

```

Comment:
There should be error handling for the case when `fromCode` does not find a matching operation. This could throw an `IllegalArgumentException` which should be caught and handled appropriately.